src/components/common/Paragraph.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Paragraph: React.FC = ({ children }) => (
+  <p className="pc:text-lg desktop:text-xl pl-10 pt-3 pr-4 pc:pl-20 pc:pt-5 pc:pr-8">{children}</p>
```

Comment:
![image](https://user-images.githubusercontent.com/34268371/137851335-f1859223-87f8-4147-84cc-527f88569b1d.png)

モバイルで左側のpaddingが多いのが気になります。左右で揃えてしまっていい気がします

```suggestion
  <p className="pc:text-lg desktop:text-xl px-4 pt-3 pc:pl-20 pc:pt-5 pc:pr-8">{children}</p>
</p>
```